src/main/scala/clump/Clump.scala

```diff
@@ -67,15 +67,6 @@ object Clump {
 
   def collect[T](clumps: List[Clump[T]]): Clump[List[T]] =
     new ClumpCollect(clumps)
-
-  def sourceFrom[T, U](fetch: Set[T] => Future[Map[T, U]], maxBatchSize: Int = Int.MaxValue) =
```

Comment:
Also at some point I was gonna try going through and removing a lot of the newlines in Clump.scala because a lot of the methods would fit comfortably on one line. This might make things look less cluttered in there